en/docs/references/preview-features.md

```diff
@@ -0,0 +1,16 @@
+# Preview Features
+
+Preview features offer optional functionality that you can enable if required.
```

Comment:
Is it correct to say optional functionality?
Preview features are more like a new capability made available for early access before it is finalized and pushed for use right?